bentoml/bentos.py

```diff
@@ -420,7 +420,7 @@ def containerize(
     buildx.health()
 
     bento = _bento_store.get(tag)
-    if docker_image_tag is None:
+    if not docker_image_tag:
         docker_image_tag = str(bento.tag)
```

Comment:
What if `docker_image_tag` is a list here?

Reply:
if `docker_image_tag` is a list, then we don't generate a image using bento.tag